Makefile.am

```diff
@@ -74,6 +74,7 @@ handwritten_tests = \
 	t/geturl_ssl.pl \
 	t/is-and-extract-ipv4.pl \
 	t/is-and-extract-ipv6.pl \
+	t/is-and-extract-ipv6-gua.pl \
```

Comment:
Rename the file to `t/is-and-extract-ipv6-global.pl`.

Reply:
Done